### PR TITLE
Adding ponies.

### PR DESCRIPTION
**Pull request by jwpeterson**

### CODE REVIEW

File: include/geom/cell_pyramid.h [hunk 1]
```diff
@@ -1,153 +1,153 @@
...
 
-protected:
+ponies:
 
   /**
-   * Data for links to parent/neighbor/interior_parent elements.
+   * ponies.
    */
-  Elem* _elemlinks_data[6+(LIBMESH_DIM>3)];
+  ponies)];
```
Comment (roystgnr) [inline include/geom/cell_pyramid.h]: Looks like an unmatched paren here.

Comment (permcody) [inline include/geom/cell_pyramid.h]: Merge! Merge! Merge! Go Go Go

**Comment by jwpeterson** [comment]
```
../src/base/periodic_boundaries.C:19:2: error: invalid preprocessing directive #ponies
```

I'm not sure what moosebuild's problem is here, this looks correct...

File: include/geom/cell_pyramid.h (hunk 1 above)
Comment (jwpeterson) [inline include/geom/cell_pyramid.h]: Good catch!

**Comment by dmcdougall** [comment]
Would moose, instead of ponies, have been more appropriate?

File: include/geom/elem.h [hunk 2]
```diff
@@ -1,2231 +1,2231 @@
...
-   * 1) operator*
-   * 2) operator++
-   * 3) operator==
-   * The definition can be found at the end of this header file.
+   * ponies
+   * ponies
+   * ponies
+   * ponies
+   * ponies*
+   * ponies++
```
Comment (permcody) [inline include/geom/elem.h]: More ponies!

File: include/geom/elem.h [hunk 3]
```diff
@@ -1,2231 +1,2231 @@
...
-inline
-Elem* Elem::child_neighbor (Elem* elem) const
+ponies
+ponies
 {
-  for (unsigned int n=0; n<elem->n_neighbors(); n++)
-    if (elem->neighbor(n) &&
-        elem->neighbor(n)->parent() == this)
-      return elem->neighbor(n);
+  ponies++)
```
Comment (permcody) [inline include/geom/elem.h]: Even more ponnies!!!

**Comment by jwpeterson** [comment]
> Would moose, instead of ponies, have been more appropriate?

Good suggestion, let me give that a try.

**Comment by dschwen** [comment]
:horse:?

**Pull request closed without merging**